### DISCUSSION TITLE
the emoji & people selection menus appear at the wrong time.

### DISCUSSION THREAD

**Discussion by 3m4r5** [community/community#18231] [Mobile] [answered]
not sure if [this](https://user-images.githubusercontent.com/63432932/173013083-cef21816-f878-4cf3-92ce-d01a043a3514.MOV) is a bug or intended behavior.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the report — it looks like it's in between being a bug and expected behavior. Especially since there are lots of different usernames someone might have, it feels safe to assume that if someone is typing `@` that we might want to populate a user suggestion list. But in this example, where you've split an email, I can see how it feels janky.

**Reply by 3m4r5** [reply]
I think showing the menu only if the cursor is after the @ or : might solve it.
